packages/relay-compiler/core/RelayCompilerError.js

```diff
@@ -110,7 +110,7 @@ function createCombinedError(
     `${prefix}Encountered ${errors.length} error(s):\n` +
       errors
         .map(error =>
-          String(error)
+          (error instanceof GraphQLError ? printError(error) : String(error))
```

Comment:
Use our own formatter for `GraphQLError` instances, instead of the one from graphql-js.